static/js/tour.js

```diff
@@ -51,30 +51,22 @@ function startTour() {
                 title: "Let's try it!",
                 content: "For now, press the \"Example\" button to preload the website with an example VCF.",
                 reflex: true,
-                template: templateWithoutNext
+                // template: templateWithoutNext
```

Comment:
These commented out sections can be deleted as well as the templateWithoutNext template. I think it is wise of you to make all sections of the tour have a next button.